test/test_rack_server.rb

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+# frozen_string_literal: true
+
+require "test/unit"
+require "rack/test"
+require "xmlrpc/server"
+require 'xmlrpc/create'
+require 'xmlrpc/parser'
+require 'xmlrpc/utils'
+
+module TestXMLRPC
+class Test_Rack < Test::Unit::TestCase
+  include Rack::Test::Methods
+  include XMLRPC::ParserWriterChooseMixin
```

Comment:
Can we use `XMLRPC::Create.new(XMLRPC::Config.default_writer.new)` and `XMLRPC::Config.default_parser.new` instead?